helpers/dbload.ts

```diff
@@ -7,26 +7,22 @@ import { ChallengeTypes, Challenge } from './models/Challenge'
 import { AlertTypes, Alert } from './models/Alert'
 import { StarTypes, Star } from './models/Star'
 
-const options: Options = {
-  host: process.env.DB_HOST || 'city',
-  logging: false,
-  dialect: 'postgres',
-  pool: {
-    max: 5,
-    min: 0,
-    acquire: 30000,
-    idle: 10000
-  }
-}
-
-if (process.env.NODE_ENV != 'production') {
-  options.port = ((process.env.DB_PORT as unknown) as number) || 5432
-}
 const sequelize = new Sequelize(
-  process.env.DB_NAME || 'you',
-  process.env.DB_USER || 'failed',
-  process.env.DB_PW || 'this',
-  options
+  process.env.DB_NAME as string,
+  process.env.DB_USER as string,
+  process.env.DB_PW,
+  {
+    host: process.env.DB_HOST,
+    logging: false,
+    port: (process.env.DB_PORT as unknown) as number,
```

Comment:
since  fields in the process.env object are strings, it will be clear if you explicitly convert the string to a number with parseInt, instead of implicitly doing it with typescript

Reply:
I still need type assertion for `parseInt`, because it requires string as argument, so it's going to be `parseInt(process.env.DB_PORT as string)`. Should I change it? I don't think there is a clear solution to this.

Reply:
Nice yea, `parseInt(process.env.DB_PORT as string)` looks about as clear as it can get

Reply:
Changed.